packages/flutter/lib/src/material/theme.dart

```diff
@@ -161,6 +161,48 @@ class Theme extends StatelessWidget {
         .resolveFrom(context);
   }
 
+  /// Retrieves the [Brightness] to use for descendant widgets, based
+  /// on the value of [ThemeData.brightness] in the given [context].
+  ///
+  /// If no [InheritedTheme] can be found in the given [context], or its `brightness`
+  /// is null, it will fall back to [MediaQueryData.platformBrightness].
+  ///
+  /// Throws an exception if no valid [InheritedTheme] or [MediaQuery] widgets
+  /// exist in the ancestry tree.
+  ///
+  /// See also:
+  ///
+  /// * [maybeBrightnessOf], which returns null if no valid [InheritedTheme] or
+  ///   [MediaQuery] exists, instead of throwing.
+  /// * [ThemeData.brightness], the property takes precedence over
+  ///   [MediaQueryData.platformBrightness] for descendant widgets.
```

Comment:
I would keep the mention of `Material`, since `Theme` is a `Material` thing and not under the widgets library.

```suggestion
  ///   [MediaQueryData.platformBrightness] for descendant Material widgets.
```